### DISCUSSION TITLE
Save YT video upload date in html file name.

### DISCUSSION THREAD

**Discussion by mataka123** [gildas-lormeau/SingleFile#1388] [Q&A]
I use SingleFile to save comment section from youtube videos. It would be nice to have the upload date in html file names so i can have them in line with my yt videos archive with also have upload date at beginning of video names.

**Comment by gildas-lormeau** [comment]
This is possible with the following file name template:
`%if-empty<{page-title}|No title> (%if-empty<%page-element-attribute<meta[itemprop="datePublished"]|content>|{date-locale} {time-locale}>).{filename-extension}`

**Comment by mataka123** [comment]
> This is possible with the following file name template: `%if-empty<{page-title}|No title> (%if-empty<%page-element-attribute<meta[itemprop="datePublished"]|content>|{date-locale} {time-locale}>).{filename-extension}`

Thank you for answer. I use now: %if-empty<%page-element-attribute<meta[itemprop="datePublished"]|content>> %if-empty<{page-title}|No title> .{filename-extension} . I receved file named as: 2023-04-10T12_00_21-07_00 Morgan Fisher - シャルロッテンブルグの夜明け (1992) - YouTube .html . Can i get rid of "T12_00_21-07_00" and " - YouTube" from file name?

**Comment by gildas-lormeau** [comment]
Sorry for the late response. You can use `%substring` to remove the date, for example `%if-empty<%substring<%page-element-attribute<meta[itemprop="datePublished"]|content>|0|10>> %if-empty<{page-title}|No title> .{filename-extension}`
